group1/comm/templates/comm/index.html

```diff
@@ -144,7 +144,11 @@ <h4 class="sidebar_description">Quick links</h4>
               </button>
                 <ul class="dropdown-menu dropdown-menu-right" role="menu">
                   <li><a href="#">Upload File</a></li>
-                  <li><a href="#">Download File</a></li>
+                  <!--Currently the file id is hardcoded into the url that is used when sending the
```

Comment:
This comment pertains to the current state of the application?  Meaning, once we demo this, we'll need to find a dynamic method of passing the file iD?

Reply:
Yes that is correct @danbudris.  I didn't want to hardcode this value without commenting on it.  If we keep the Google Drive functionality implemented in the python backend, we will some way to get the file id from the user for download.  The Google Picker functionality (https://developers.google.com/picker/) that @nishgaddam is working on will allow the user to choose a file from their Drive after they authenticate with us and we can use that for the file id to download.  If we move the Google Drive entirely to the frontend javascript it might be easier to access the authentication credentials and whatever other information we need for file download, file upload, and file share.  We can talk about that after this presentation.